dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/AuthorRole.cs

```diff
@@ -31,6 +31,11 @@ namespace Microsoft.SemanticKernel.AI.ChatCompletion;
     /// </summary>
     public static readonly AuthorRole Tool = new("tool");
 
+    /// <summary>
+    /// The role that provides information about a function call result.
+    /// </summary>
+    public static readonly AuthorRole Function = new("function");
```

Comment:
All of this System/Assistant/User/Tool/Function fields should be changed to properties, e.g.
```C#
public static AuthorRole Function { get; } = new("function");
```